app/routes/circulars._archive._index/route.tsx

```diff
@@ -68,7 +78,15 @@ export async function loader({ request: { url } }: LoaderFunctionArgs) {
     sort,
   })
   const requestedChangeCount = (await getChangeRequests()).length
-  return { page, ...results, requestedChangeCount }
+
+  return {
+    page,
+    ...results,
+    requestedChangeCount,
+    synonymFlagIsOn,
```

Comment:
No need to pass this through the loader. This value is already available in components through the `useFeature` hook.